how-to-docs/ingress.md

```diff
@@ -0,0 +1,36 @@
+# Using Ingress
+
+An Ingress is a type of Kubernetes resource that allows you to expose your services outside the cluster it gets deployed and managed exactly like other Kube resources.
+
+Our ingress setup offers two different ingresses based on how restrictively you want to expose your services:
+- internal - only people within the VPN can access services
+- external - anyone with internet access can access services
+
+The annotation ```kubernetes.io/ingress.class: "nginx-internal"``` is used to specify whether the ingress is internal. (```kubernetes.io/ingress.class: "nginx-external"``` is used for an external ingress.)
```

Comment:
Note you can also just use a single backtick when you have code inline like this. (I didn't even know 3 backticks were allowed inline! :))